apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -56,6 +60,9 @@ type DatabaseSpec struct {
 
 	// Defines the Name of the app to share a database from
 	SharedDBAppName string `json:"sharedDbAppName,omitempty"`
+
+	// T-shirt size, one of small, medium, large
+	DBTShirtSize TShirtSize `json:"dbTShirtSize,omitempty"`
```

Comment:
I am nitpicking here. But could we drop `TShirt` from all the naming to make the field names and code a little cleaner? :) For example could this just be `dbSize` or `volumeSize`? 

Could `TShirtSizeDefinitions` just become `SizeDefinitions`?

Reply:
I went for `TShirt` because we are using the small, medium, large terminology.